src/components/atoms/DeviceVideo/DeviceVideo.tsx

```diff
@@ -18,6 +18,7 @@ export const DeviceVideo: React.FC<DeviceVideoProps> = ({
   const [error, setError] = useState();
   const [playing, setPlaying] = useState(false);
   const [mediaStream, setMediaStream] = useState<MediaStream | null>(null);
+  const [cameraPermission, setCameraPermission] = useState<PermissionState>();
```

Comment:
Primarily used to display in browser console if/when permission was granted. Ideally this can be of use for later addition of nicer UI messages about the nature of the stream (allowed/denied/error)